website/docs/r/google_project_default_service_accounts.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+subcategory: "Cloud Platform"
+layout: "google"
+page_title: "Google: google_project_default_service_accounts"
+sidebar_current: "docs-google-project-default-service-accounts-x"
+description: |-
+  Allows management of Google Cloud Platform project default service accounts.
+---
+
+# google_project_default_service_accounts
+
+Allows management of Google Cloud Platform project default service accounts.
+
+When certain service APIs are enabled, Google Cloud Platform automatically creates service accounts to help get started, but
+this is not recommended for production environments as per [Google's documentation](https://cloud.google.com/iam/docs/service-accounts#default).
+See the [Organization documentation](https://cloud.google.com/resource-manager/docs/quickstarts) for more details.
+
+## Example Usage
+
+```hcl
+resource "google_project_default_service_accounts" "my_project" {
+  project = "my-project-id"
+  action = "DELETE"
+}
+```
+
+To enable the default service accounts on the resource destroy:
+
+```hcl
+resource "google_project_default_service_accounts" "my_project" {
+  project = "my-project-id"
+  action = "DISABLE"
+  restore_policy = "REACTIVATE"
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `project` - (Required) The project ID. Changing this forces the actions to be applied in the new project.
+
+- `action` - (Optional) The action to be performed in the default service accounts. Valid values are: deprivilege, delete, disable.
```

Comment:
Make sure the valid values match the user-specifiable ones exactly. I'd also recommend backticks to `code font` the values.